app/views/layouts/application.haml

```diff
@@ -12,7 +12,7 @@
   = javascript_include_tag 'application', 'data-turbolinks-track' => true
   = csrf_meta_tags
   = render 'layouts/shim'
-%body
+%body.container{'data-spy' => "scroll", 'data-target' => ".navbar"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace